tests/unit_tests/device/hip/CMakeLists.txt

```diff
@@ -7,6 +7,8 @@
 message(STATUS "Building detray HIP unit tests")
 cmake_minimum_required(VERSION 3.21) # HIP langauge support requires minimum 3.21
 
+find_package(HIPToolkit)
```

Comment:
These changes will also need to be made to the other HIP cmake files @Larb-YasmineBadrElhouda . Including the renaming the file that contains the actual test from  `.hip` to `.cpp`